Makefile

```diff
@@ -8,7 +8,7 @@ build:
 	docker tag ${IMG} ${LATEST}
 
 test:
-	docker run ${NAME} /bin/sh -c "python manage.py test"
+	docker run ${NAME} /bin/sh -c "python manage.py test ilmo_app/ palautteet/tests"
```

Comment:
The former command should have already ran all the tests in every app if they are named correctly.